sdv/constraints/tabular.py

```diff
@@ -405,6 +405,7 @@ def __init__(self, low_column_name, high_column_name, strict_boundaries=False):
         self.constraint_columns = (low_column_name, high_column_name)
         self._dtype = None
         self._is_datetime = None
+        self.nan_column_name = None
```

Comment:
minor: can we make this private ie. (`_nan_column_name`)

Reply:
Yes done in [5c8aa02](https://github.com/sdv-dev/SDV/pull/1399/commits/5c8aa0255bf6e7bbc870ce905c75c43a4e6cd507)